src/BagReader.ts

```diff
@@ -60,29 +66,32 @@ export default class BagReader {
   // generally you call this first
   // because you need the header information to call readConnectionsAndChunkInfo
   readHeader(callback: Callback<BagHeader>) {
-    this.verifyBagHeader(callback, () => {
-      return this._file.read(HEADER_OFFSET, HEADER_READAHEAD, (error: Error | null, buffer?: Buffer) => {
+    this.verifyBagHeader(callback, () =>
```

Comment:
This was because of the `arrow-body-style` rule, however it turns out the returns were `void`, and so nothing needed to be returned. Rewrote the returns here to make this clear and maintain the brackets.